### PR TITLE
added WORDS_TO_DWORD.fct under assembling directory in utils typelib

### PR DESCRIPTION
**Pull request by MoritzO02**
Also added assembling directory under utils-typelib
for issue #221

### CODE REVIEW

File: data/typelibrary/utils-1.0.0/typelib/assembling/WORDS_TO_DWORD.fct [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	IN_00 : WORD;    // WORD 00
+	IN_01 : WORD;    // WORD 01
+END_VAR
+
+VAR_OUTPUT
+	OUT : DWORD;    // DWORD Output
+END_VAR
+
+(* this Function combines the 2 WORDS to a dword *)
+OUT := SHL(WORD_TO_DWORD(IN_01), SHIFT_WORD_01)
```
Comment (MartinMelikMerkumians) [inline data/typelibrary/utils-1.0.0/typelib/assembling/WORDS_TO_DWORD.fct]: you can simplify this code to two .$Wx access operator assignments

**Review by MartinMelikMerkumians — CHANGES_REQUESTED** [review]
Please propose a different name as we reserved the *_TO_** function names for official standardized casts

File: data/typelibrary/utils-1.0.0/typelib/assembling/WORDS_TO_DWORD.fct (hunk 1 above)
Comment (diplfranzhoepfinger) [reply data/typelibrary/utils-1.0.0/typelib/assembling/WORDS_TO_DWORD.fct]: https://engineer.plcnext.help/2024.0_LTS_de/_index.htm?iframe=https://engineer.plcnext.help/2024.0_LTS_de/Variables_AnyBit_Access.htm 

is it this ? 


and then is it .%Wx or .$Wx ?

File: data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_WORDS.fct [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+		<ST><![CDATA[FUNCTION ASSEMBLE_DWORD_FROM_WORDS : DWORD
+
+VAR_INPUT
+	IN_00 : WORD;    // WORD 00
+	IN_01 : WORD;    // WORD 01
+END_VAR
+
+// VAR_OUTPUT
+// OUT : DWORD; // DWORD Output
+// END_VAR
```
Comment (diplfranzhoepfinger) [inline data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_WORDS.fct]: remove this please, it gives no Information. 
please remove all 3 lines. 
@MoritzO02

File: data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_WORDS.fct [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+(* this Function combines the 2 WORDS to a dword *)
+ASSEMBLE_DWORD_FROM_WORDS.%W0 := IN_00;
```
Comment (diplfranzhoepfinger) [inline data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_WORDS.fct]: this is cool Syntax @MartinMelikMerkumians

**Review by diplfranzhoepfinger — APPROVED** [review]
looks good to me, Approved

**Review by franz-hoepfinger-4diac — APPROVED** [review]
Approved

**Review by franz-hoepfinger-4diac — APPROVED** [review]

**Review by azoitl — APPROVED** [review]

**Pull request merged**